### ISSUE TITLE
PSIO

### ISSUE THREAD

**Issue by GoobyCorp** [Lameguy64/PSn00b-Debugger#1]
You're working with the PSIO team to implement this on their hardware?

**Comment by Lameguy64** [comment]
Yes. They wanted my debug monitor in PSIO to give it better development capabilities. Haven't seen how they implemented it yet aside from some screenshots of their own PC side debugger interface.

**Comment by GoobyCorp** [comment]
That's awesome, I was "looking" through the system app source and saw your name mentioned. I wonder if they could somehow load in a development kernel considering the parallel IO port has DMA.

**Comment by GoobyCorp** [comment]
I thought I might as well ask this here, how does one go about initiating a disc swap with your SDK?

**Comment by Lameguy64** [comment]
I'm not so sure how disc swap is done with the BIOS CD-ROM functions and there does not appear to be functions in the BIOS that would be useful for such operations. Maybe it is automatic and you just have to check a file after a disc swap?

**Comment by GoobyCorp** [comment]
The action replay source leaked a while back and it says something about _96_remove then cd_init for swapping discs, no idea where those are in the kernel though.

**Comment by Lameguy64** [comment]
_96_init is often confused with cd_init. I guess it basically just uninstalls and reinstalls the BIOS CD-ROM subsystem on disc swap.
